recipes/netpbm/meta.yaml

```diff
@@ -20,7 +20,6 @@ requirements:
     - libpng
     - zlib
     - lxml
-    - perl
```

Comment:
Don't you need it? If not, removal is fine. Though it looks like the conda-build-all issue is fixed so should be fine having it again.